test/infrastructure/docker/internal/docker/manager.go

```diff
@@ -106,12 +109,20 @@ func (m *Manager) CreateWorkerNode(ctx context.Context, name, clusterName string
 // This can break the Kubeconfig in kind, i.e. the file resulting from `kind get kubeconfig -n $CLUSTER_NAME' if the load balancer container is restarted.
 func (m *Manager) CreateExternalLoadBalancerNode(ctx context.Context, name, image, clusterName, listenAddress string, port int32, _ clusterv1.ClusterIPFamily) (*types.Node, error) {
 	// load balancer port mapping
-	portMappings := []v1alpha4.PortMapping{{
-		ListenAddress: listenAddress,
-		HostPort:      port,
-		ContainerPort: ControlPlanePort,
-		Protocol:      v1alpha4.PortMappingProtocolTCP,
-	}}
+	portMappings := []v1alpha4.PortMapping{
+		{
+			ListenAddress: listenAddress,
+			HostPort:      port,
+			ContainerPort: ControlPlanePort,
+			Protocol:      v1alpha4.PortMappingProtocolTCP,
+		},
+		{
+			ListenAddress: listenAddress,
+			HostPort:      port,
```

Comment:
```suggestion
			HostPort:      0,
```

Let's always pick a port. passing in two times the same port != 0 should lead to issues.